.github/actions/run-backend-tests/action.yml

```diff
@@ -12,7 +12,7 @@ inputs:
         description: ClickHouse server image tag, e.g. clickhouse/clickhouse-server:latest
     segment:
         required: true
-        description: Either 'FOSS' or 'EE' segment
+        description: Either 'FOSS' or 'TEMPORAL' segment
```

Comment:
"FOSS" no longer makes sense, should just be "Core" and "Temporal"

Reply:
reasonable 👍